todo/src/actions.js

```diff
@@ -0,0 +1,46 @@
+
+export function addTodo(todo,important) {
+    // console.log('addTodo action maker has been triggered ', todo);
+    return {
+        type: "ADD_TODO",
+        payload: {
+            todo : todo,
+            important : important
+        }
+    }
+}
+
+export function deleteTodo(todo,i) {
+    console.log("deleteTodo action creator has been triggered ", todo, i);
+    return {
+        type: "DELETE_TODO",
+        payload: {
+            todo : todo,
+            index : i
+        }
```

Comment:
You  are not utilizing the Todo itself, only the index in your reducer  so there's is no need to pass the Todo into the method here